go-fuzz/stats.go

```diff
@@ -0,0 +1,167 @@
+            <span class="text-muted">Crashers</span>
+          </div>
+          <div class="col-xs-3 col-sm-1 placeholder">
+            <h4 id="restarts"></h4>
+            <span class="text-muted">Restarts</span>
+          </div>
+          <div class="col-xs-4 col-sm-1 placeholder">
+            <h4 id="execs"></h4>
+            <span class="text-muted">Execs</span>
+          </div>
+          <div class="col-xs-4 col-sm-1 placeholder">
+            <h4 id="cover"></h4>
+            <span class="text-muted">Cover</span>
+          </div>
+          <div class="col-xs-4 col-sm-1 placeholder">
+            <h4 id="uptime"></h4>
+            <span class="text-muted">Uptime</span>
+          </div>
+        </div>
+
+        <h2 class="sub-header">History</h2>
+        <div class="table-responsive">
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th>Slaves</th>
+                <th>Corpus</th>
+                <th>Crashers</th>
+                <th>Restarts</th>
+                <th>Execs</th>
+                <th>Cover</th>
+                <th>Uptime</th>
+              </tr>
+            </thead>
+	    <tbody></tbody>
+	  </table>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+
+
+<!-- The following <style> and <script> blocks work around a specific bug with width
+in IE 10 in Windows 8 and Windows Phone 8. See:
+http://getbootstrap.com/getting-started/#support-ie10-width -->
+<style>
+@-webkit-viewport   { width: device-width; }
+@-moz-viewport      { width: device-width; }
+@-ms-viewport       { width: device-width; }
+@-o-viewport        { width: device-width; }
+@viewport           { width: device-width; }
+</style>
+<script>
+// Copyright 2014-2015 Twitter, Inc.
+// Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
+if (navigator.userAgent.match(/IEMobile\/10\.0/)) {
+	var msViewportStyle = document.createElement('style')
+	msViewportStyle.appendChild(
+		document.createTextNode(
+			'@-ms-viewport{width:auto!important}'
+		)
+	)
+	document.querySelector('head').appendChild(msViewportStyle)
+}
+</script>
+
+<script>
+// First, checks if it isn't implemented yet.
+if (!String.prototype.format) {
+	String.prototype.format = function() {
+		var args = arguments;
+		return this.replace(/{(\d+)}/g, function(match, number) { 
+			return typeof args[number] != 'undefined'
+			? args[number]
+			: match
+			;
+		});
+	};
+}
+
+var rowFmt = "<tr><td>{0}</td><td>{1}</td><td>{2}</td><td>{3}</td><td>{4}</td><td>{5}</td><td>{6}</td></tr>"
+
+var evtSource = new EventSource("/eventsource");
+evtSource.addEventListener("ping", function(e) {
+	var data = JSON.parse(e.data);
+	var uptime = formatDuration(Date.now() - Date.parse(data.StartTime))
```

Comment:
data.StartTime is time on server running go-fuzz
Date.now() is local time on client machine.
The difference is meaningless.
Also, I don't want to maintain long javascript functions.
Let's just push formatted uptime from server (we already have format function there).
If we ever need raw uptime in seconds and/or server start time/current time, we can always add them later.